optimum/habana/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -434,3 +517,126 @@ def custom_forward(*inputs):
         attentions=all_self_attentions,
         cross_attentions=all_cross_attentions,
     )
+
+
+class GaudiGPT2LMHeadModel(GPT2LMHeadModel):
+    def __init__(self, config):
+        super().__init__(config)
+        self.lm_head_chunks = []
```

Comment:
Is this used anywhere? Or is it for a future PR?

Reply:
not used code, has been removed